lib/components/viewers/viewers.css

```diff
@@ -273,3 +273,59 @@ a.routeDetails {
   float: right;
   width: 50px;
 }
+
+/* Route Viewer Updates */
+.search-and-filter {
+  margin-top: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+}
+.search-and-filter select {
+  margin-left: 10px;
+  text-overflow: ellipsis;
+  min-width: 105px;
```

Comment:
Is this one useful?

Reply:
Yes otherwise the dynamic spacing cuts the text